### PR TITLE
feat(3000): Add Data Management to the sidebar

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

Event/property definitions and actions in the sidebar:

Screenshot 2023-06-12 at 17 33 15

### CODE REVIEW

File: frontend/src/layout/navigation-3000/sidebars/personsAndGroups.ts [hunk 1]
```diff
@@ -170,6 +169,14 @@ export const personsAndGroupsSidebarLogic = kea<personsAndGroupsSidebarLogicType
         // kea-typegen doesn't like selectors without deps, so searchTerm is just for appearances
         debounceSearch: [(s) => [s.searchTerm], () => true],
     })),
+    listeners(({ cache }) => ({
+        loadPersons: async ({ url }) => {
+            const offset = url ? parseInt(new URL(url).searchParams.get('offset') || '0') : 0
+            if (offset === 0) {
+                cache.requestedPersons.length = 0 // Clear cache
```
Comment (thmsobrmlr) [inline frontend/src/layout/navigation-3000/sidebars/personsAndGroups.ts:176]: 🤔
```suggestion
                cache.requestedPersons = [] // Clear cache
```

File: frontend/src/layout/navigation-3000/sidebars/insights.ts [hunk 2]
```diff
@@ -136,6 +136,13 @@ export const insightsSidebarLogic = kea<insightsSidebarLogicType>([
         // kea-typegen doesn't like selectors without deps, so searchTerm is just for appearances
         debounceSearch: [(s) => [s.searchTerm], () => true],
     })),
+    listeners(({ values, cache }) => ({
+        loadInsights: () => {
+            if (!values.paramsFromFilters.offset) {
+                cache.requestedInsights.length = 0
```
Comment (thmsobrmlr) [inline frontend/src/layout/navigation-3000/sidebars/insights.ts:142]: 🤔 is there anything that prevents a reset like this:

```suggestion
                cache.requestedInsights = []
```

**Review by thmsobrmlr — APPROVED** [review]
Looks good!

Left two comments inline.

File: frontend/src/layout/navigation-3000/sidebars/personsAndGroups.ts (hunk 1 above)
Comment (Twixes) [reply frontend/src/layout/navigation-3000/sidebars/personsAndGroups.ts:176]: Just performance, `.length = 0` is the fastest way to do this (weirdly JS has no `.clear()`, but this does the same as `.clear()` in Python)

**Pull request merged**